cfme/tests/openstack/cloud/test_volumes.py

```diff
@@ -0,0 +1,189 @@
+"""Tests for cloud provider's volumes"""
+
+import pytest
+from fauxfactory import gen_alphanumeric
+from cfme.cloud.instance import Instance
+from cfme.cloud.volume import (creation_form, device_input, get_volume_name,
+                               list_tbl, select_vm, select_volume, Volume)
+from cfme.fixtures import pytest_selenium
+from cfme.web_ui import InfoBlock, toolbar
+from cfme.web_ui.flash import assert_message_contain
+from cfme.web_ui.form_buttons import (attach_volume, detach_volume, FormButton,
+                                      submit_changes)
+from random import choice
+from utils import testgen, version
+from utils.appliance.implementations.ui import navigate_to
+from utils.wait import wait_for
+
+pytest_generate_tests = testgen.generate(testgen.provider_by_type,
+                                         ['openstack'], scope='module')
+
+
+PROD_VERSION = version.current_version().product_version()
```

Comment:
nope nope nope, this command goes to the appliance on import of this module which we don't want. Put the version version check resolution in the lambda where it is used and do not use the product_version but just the version instead